iib/workers/tasks/build.py

```diff
@@ -620,20 +624,19 @@ def handle_add_request(
 
     set_request_state(request_id, 'in_progress', 'Creating the manifest list')
     output_pull_spec = _create_and_push_manifest_list(request_id, arches)
-
-    if legacy_support_packages:
-        export_legacy_packages(
-            legacy_support_packages, request_id, output_pull_spec, cnr_token, organization
-        )
-
-    _finish_request_post_build(
+    modified_index_image = update_index_image_pull_spec(
         output_pull_spec,
         request_id,
         arches,
         from_index,
         overwrite_from_index,
         overwrite_from_index_token,
     )
+    if legacy_support_packages:
+        export_legacy_packages(
+            legacy_support_packages, request_id, modified_index_image, cnr_token, organization
+        )
+    set_request_state(request_id, 'complete', 'Operator bundle added to index image')
```

Comment:
How about adding `The request has been completed successfully.` to the state_reason here.

Reply:
There can be multiple bundles here. Could we please change `bundle` to `bundles(s)`

Reply:
I agree about the bundle(s), I changed the completion message assuming that stating what the request did finally might be more meaningful then the generic completion message. However, I might be wrong about it and I wouldn't mind using the above-mentioned message.

Reply:
yup, you're right. Adding more specific info is more helpful. I meant appending the string `The request has been completed successfully.` to the existing message that you have.